src/main/java/duke/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,59 @@
+package duke.command;
+import duke.exception.DukeException;
+import duke.ui.Ui;
+import duke.task.Deadline;
+import duke.task.TaskList;
+/**
+ * It is a command object extends from Command for the Duke program.
+ * When the parser calls it, it will receive the requests from the users
+ * during the running of the program and starts add new deadlines to the task list.
+ */
+public class AddDeadlineCommand extends Command {
+
+    /**
+     * Constructor for AddDeadlineCommand object
+     *
+     * @param userMessage The message that the user inputs for further execution.
+     */
+    public AddDeadlineCommand(String userMessage) {
+        super(userMessage);
+    }
+
+    /**
+     * The execution after parsing, it will add a deadline object into the tasks.
+     * If the input is not correct, it will raise an exception.
```

Comment:
Perhaps you can phrase your JavaDoc comment in a different way, such as "**Adds** a Deadline object into the tasks after parsing of execution" and "**Raises** an exception ..."

Reply:
Agree